tests/openssl/README.md

```diff
@@ -0,0 +1,14 @@
+OpenSSL tests
+=============
+
+This directory includes the unit tests from OpenSSL (under `openssl/test`). The tests are partitioned
+into three categories as follows.
+
+* tests.supported.buildtest - The tests generated by the openssl/test/generate_buildtest.pl script.
+* tests.supported - The rest of tests that are supported by OE.
+* tests.unsupported - The tests that are currently not supported by OE.
+
+The `test_data` directory includes a subset of files required by the tests. These files are required
```

Comment:
Thanks for the explanation here ... I'm very reluctant to handle it this way though since it's possible for these files to get out of sync. Have you considered using the CMake configure_file command as part of the build step for the tests to copy the source files directly to the test folders and assert a specific line ending? i.e. with the [`NEWLINE_STYLE LF`](https://cmake.org/cmake/help/v3.13/command/configure_file.html) option

Reply:
After some investigations, here are some results:
- the `NEWLINE_STYLE LF` option does not work properly as it always insert a new line and the end of the newly created file. This behavior still causes some tests to fail.
- I tried to use `file(READ ...)` `string(REGEX REPLACE ...)` `file(WRITE ...)` to replace CR (`\r`), but it does not work 😢 
- Finally, I found the `dos2unix` utility that comes with the git, which works nicely. So I end up using this approach to convert CRLF to LF on Windows.